### ISSUE TITLE
Error when accessing Trades

### ISSUE THREAD

**Issue by klara31** [freqtrade/freqtrade#4405]
**Describe the bug**
When accessing the trades, freqtrade generates an error (see below). I can't tell if frequi or freqtrade is to blame, the error is triggered by frequi I am using binance as exchange.

**To Reproduce**
1. Go to trades
2. Click on Pairs-combined
3. Scroll down to ADA/BTC and select
4. See error below

**Expected behavior**
The chart should be loaded, however only 'No Data' is displayed. 

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser tested with Safari and Chrome
 - Version Big Sur

**Additional context**
The log contents from freqtrade:
```
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/root/.local/lib/python3.8/site-packages/uvicorn/protocols/http/h11_impl.py", line 394, in run_asgi
    result = await app(self.scope, self.receive, self.send)
  File "/root/.local/lib/python3.8/site-packages/uvicorn/middleware/proxy_headers.py", line 45, in __call__
    return await self.app(scope, receive, send)
  File "/root/.local/lib/python3.8/site-packages/fastapi/applications.py", line 199, in __call__
    await super().__call__(scope, receive, send)
  File "/root/.local/lib/python3.8/site-packages/starlette/applications.py", line 111, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/root/.local/lib/python3.8/site-packages/starlette/middleware/errors.py", line 181, in __call__
    raise exc from None
  File "/root/.local/lib/python3.8/site-packages/starlette/middleware/errors.py", line 159, in __call__
    await self.app(scope, receive, _send)
  File "/root/.local/lib/python3.8/site-packages/starlette/middleware/cors.py", line 86, in __call__
    await self.simple_response(scope, receive, send, request_headers=headers)
  File "/root/.local/lib/python3.8/site-packages/starlette/middleware/cors.py", line 142, in simple_response
    await self.app(scope, receive, send)
  File "/root/.local/lib/python3.8/site-packages/starlette/exceptions.py", line 82, in __call__
    raise exc from None
  File "/root/.local/lib/python3.8/site-packages/starlette/exceptions.py", line 71, in __call__
    await self.app(scope, receive, sender)
  File "/root/.local/lib/python3.8/site-packages/starlette/routing.py", line 566, in __call__
    await route.handle(scope, receive, send)
  File "/root/.local/lib/python3.8/site-packages/starlette/routing.py", line 227, in handle
    await self.app(scope, receive, send)
  File "/root/.local/lib/python3.8/site-packages/starlette/routing.py", line 41, in app
    response = await func(request)
  File "/root/.local/lib/python3.8/site-packages/fastapi/routing.py", line 220, in app
    response = actual_response_class(
  File "/root/.local/lib/python3.8/site-packages/starlette/responses.py", line 49, in __init__
    self.body = self.render(content)
  File "/root/.local/lib/python3.8/site-packages/starlette/responses.py", line 157, in render
    return json.dumps(
  File "/usr/local/lib/python3.8/json/__init__.py", line 234, in dumps
    return cls(
  File "/usr/local/lib/python3.8/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/local/lib/python3.8/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
ValueError: Out of range float values are not JSON compliant
2021-02-20 09:23:18,195 - uvicorn.error - ERROR - Exception in ASGI application
Traceback (most recent call last):
  File "/root/.local/lib/python3.8/site-packages/uvicorn/protocols/http/h11_impl.py", line 394, in run_asgi
    result = await app(self.scope, self.receive, self.send)
  File "/root/.local/lib/python3.8/site-packages/uvicorn/middleware/proxy_headers.py", line 45, in __call__
    return await self.app(scope, receive, send)
  File "/root/.local/lib/python3.8/site-packages/fastapi/applications.py", line 199, in __call__
    await super().__call__(scope, receive, send)
  File "/root/.local/lib/python3.8/site-packages/starlette/applications.py", line 111, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/root/.local/lib/python3.8/site-packages/starlette/middleware/errors.py", line 181, in __call__
    raise exc from None
  File "/root/.local/lib/python3.8/site-packages/starlette/middleware/errors.py", line 159, in __call__
    await self.app(scope, receive, _send)
  File "/root/.local/lib/python3.8/site-packages/starlette/middleware/cors.py", line 86, in __call__
    await self.simple_response(scope, receive, send, request_headers=headers)
  File "/root/.local/lib/python3.8/site-packages/starlette/middleware/cors.py", line 142, in simple_response
    await self.app(scope, receive, send)
  File "/root/.local/lib/python3.8/site-packages/starlette/exceptions.py", line 82, in __call__
    raise exc from None
  File "/root/.local/lib/python3.8/site-packages/starlette/exceptions.py", line 71, in __call__
    await self.app(scope, receive, sender)
  File "/root/.local/lib/python3.8/site-packages/starlette/routing.py", line 566, in __call__
    await route.handle(scope, receive, send)
  File "/root/.local/lib/python3.8/site-packages/starlette/routing.py", line 227, in handle
    await self.app(scope, receive, send)
  File "/root/.local/lib/python3.8/site-packages/starlette/routing.py", line 41, in app
    response = await func(request)
  File "/root/.local/lib/python3.8/site-packages/fastapi/routing.py", line 220, in app
    response = actual_response_class(
  File "/root/.local/lib/python3.8/site-packages/starlette/responses.py", line 49, in __init__
    self.body = self.render(content)
  File "/root/.local/lib/python3.8/site-packages/starlette/responses.py", line 157, in render
    return json.dumps(
  File "/usr/local/lib/python3.8/json/__init__.py", line 234, in dumps
    return cls(
  File "/usr/local/lib/python3.8/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/local/lib/python3.8/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
ValueError: Out of range float values are not JSON compliant
2021-02-20 09:23:26,959 - freqtrade.worker - INFO - Bot heartbeat. PID=1, version='2021.1', state='RUNNING'
```

**Comment by klara31** [comment]
I am running freqtrade in docker on a linux host. My docker-compose.yml:
```
  freqtrade:
    image: freqtradeorg/freqtrade:stable
    restart: unless-stopped
    container_name: freqtrade
    network_mode: bridge
    ports:
      - 8081:8081
    volumes:
      - ${USERDIR}/freqtrade:/freqtrade/user_data
      - /etc/localtime:/etc/localtime:ro
    # Default command used when running `docker compose up`
    command: >
      trade
      --logfile /freqtrade/user_data/logs/freqtrade.log
      --db-url sqlite:////freqtrade/user_data/tradesv3.sqlite
      --config /freqtrade/user_data/config.json
      --strategy xxxx
```
Freqtrade version:
```
root@4e01255f5826:/freqtrade# freqtrade -V
freqtrade 2021.1
```
Please let me know if you need more info...

**Comment by xmatthias** [comment]
it seems like something in your trades becomes either infinitely small (or infinitely big).

It's quite difficult to debug too, as it's unclear which endpoint this comes from.
Are you able to use the chrome (or firefox) dev tools and use the network tab to check which call fails? (I'll only need the url i think).

I suspect it's eihter `/status` - or `/trades` ...

**Comment by klara31** [comment]
I just looped through all the pairs in the trade and all of them are working now... The typical demo effect. I did not change anything. I am on binance, and did notice some availability issues lately, as the log contains a number of message lines like ```freqtrade.data.converter - INFO - Missing data fillup for LINK/BTC: before: 999 - after: 1014```.  I do not recall if these are the same pairs having this issue. 

I will let you know as soon as this problem occurs again.
